salat-core/src/test/scala/com/novus/salat/test/BinarySpec.scala

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (c) 2010 - 2012 Novus Partners, Inc. (http://www.novus.com)
+ *
+ * Module:        salat-core
+ * Class:         CollectionSupportSpec.scala
+ * Last modified: 2012-10-15 20:40:59 EDT
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ *           Project:  http://github.com/novus/salat
+ *              Wiki:  http://github.com/novus/salat/wiki
+ *      Mailing list:  http://groups.google.com/group/scala-salat
+ *     StackOverflow:  http://stackoverflow.com/questions/tagged/salat
+ */
+package com.novus.salat.test
+
+import com.mongodb.casbah.Imports._
+import com.mongodb.casbah.commons.MongoDBObject
+import com.novus.salat._
+import com.novus.salat.dao.SalatDAO
+import com.novus.salat.test.global._
+import com.novus.salat.test.model._
+import org.junit.runner.RunWith
+import org.specs2.runner.JUnitRunner
+
+@RunWith(classOf[JUnitRunner])
```

Comment:
Side note - how did anyone debug tests in eclipse? The Specs need to be annotated like this to be debuggable